packages/backend/src/api/request.ts

```diff
@@ -197,23 +216,3 @@ function parseError(error: ClerkAPIErrorJSON): ClerkAPIError {
     },
   };
 }
-
-class ClerkAPIResponseError extends Error {
```

Comment:
❓ This was unused I guess?

Reply:
Yes it is being imported from `@clerk/shared` now!